netdeploy/remote/nodecfg/nodeConfigurator.go

```diff
@@ -134,6 +149,23 @@ func (nc *nodeConfigurator) prepareNodeDirs(configs []remote.NodeConfig, rootCon
 			return
 		}
 
+		// Copy the bootstrapped files into current ledger folder
+		if nc.bootstrappedBlockFile != "" {
+			fmt.Fprintf(os.Stdout, "... copying block database file to ledger folder ...\n")
+			_, err = util.CopyFile(nc.bootstrappedBlockFile, filepath.Join(nodeDest, genesisDir, fmt.Sprintf("%s.block.sqlite", config.LedgerFilenamePrefix)))
+			if err != nil {
+				return
```

Comment:
in case of an error, provide more detailed error - 
i.e. fmt.Errorf("failed to copy database file %s from %s to %s : %w", ... )